### PR TITLE
Documentation clarifications and adjustments to the UDP provider default attributes

### PR DESCRIPTION
**Pull request by shefty**

### CODE REVIEW

File: prov/udp/src/udpx_attr.c [hunk 1]
```diff
@@ -66,10 +66,10 @@ struct fi_domain_attr udpx_domain_attr = {
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
 	.mr_mode = FI_MR_SCALABLE,
-	.cq_cnt = (1 << 16),
+	.cq_cnt = 1024,
 	.ep_cnt = (1 << 15),
```
Comment (bturrubiates) [inline prov/udp/src/udpx_attr.c]: Should `ep_cnt` also be reduced?

**Comment by bturrubiates** [comment]
Looks like one of these commits doesn't have a signed-off-by line.

**Comment by shefty** [comment]
Added signed-off-by line to missing patch and re-pushed

**Comment by bturrubiates** [comment]
> Added signed-off-by line to missing patch and re-pushed

Cool. Other than my question about `ep_cnt`, 👍

File: prov/udp/src/udpx_attr.c (hunk 1 above)
Comment (shefty) [inline prov/udp/src/udpx_attr.c]: updated, but I guess this view doesn't reflect the changes.

I went back and forth on ep_cnt.  The tx_cxt_cnt and rx_ctx_cnt attributes are set to the lower limit.  A better option is to determine this dynamically, but figuring that out seemed like too much work.

**Pull request merged**